solr/modules/ltr/src/test/org/apache/solr/ltr/model/TestAdapterModel.java

```diff
@@ -62,21 +62,22 @@ public void setup() throws Exception {
         "popularity", FieldValueFeature.class.getName(), "test", "{\"field\":\"popularity\"}");
 
     scoreValue = random().nextFloat();
-    final File scoreValueFile = new File(tmpConfDir, "scoreValue.txt");
+    final Path scoreValueFile = tmpConfDir.resolve("scoreValue.txt");
     try (BufferedWriter writer =
         new BufferedWriter(
-            new OutputStreamWriter(new FileOutputStream(scoreValueFile), StandardCharsets.UTF_8))) {
+            new OutputStreamWriter(
+                Files.newOutputStream(scoreValueFile), StandardCharsets.UTF_8))) {
       writer.write(Float.toString(scoreValue));
     }
-    scoreValueFile.deleteOnExit();
+    scoreValueFile.toFile().deleteOnExit();
```

Comment:
I suspect we don't need this because the `tmpConfDir` should clean itself up after the test, but we don't have to address that here/now.

Reply:
addressed in d1c7ce9